interop/client/client.go

```diff
@@ -52,6 +52,7 @@ var (
 	caFile                                 = flag.String("ca_file", "", "The file containning the CA root cert file")
 	useTLS                                 = flag.Bool("use_tls", false, "Connection uses TLS if true")
 	useALTS                                = flag.Bool("use_alts", false, "Connection uses ALTS if true (this option can only be used on GCP)")
+	useWithBlock                           = flag.Bool("use_with_block", true, "Whether or not to use the WithBlock() option when dialing")
```

Comment:
Default to false?

Reply:
Disabled since it shouldn't actually be needed now.

Reply:
+1 for just deleting it.  Not for this PR but maybe we should officially mark this as `Deprecated`?  It seems like people like using it, but the other languages don't have it and in light of idleness and balancers like ring hash, it can be a bit awkward.